src/app/containers/App/App.test.jsx

```diff
@@ -185,6 +185,7 @@ describe('App', () => {
           expect(route.getInitialData).toHaveBeenCalledWith({
             path: pathname,
             service: 'news',
+            variant: 'simp',
```

Comment:
Is it worth changing the variant to `default`, or the service to one that has a `simp` variant, so that this matches the actual data that we have?

Reply:
https://github.com/bbc/simorgh/blob/61361b85439cbaa8c11107a6552cfb998ade7e88/src/app/containers/App/App.test.jsx

so it's expecting simp because we have added it here to have variant of `/simp` 🤦. What I'm going to do is change it to a service with a variant so the tests make more sense.